diff --git a/x86_64/corefreqk.c b/x86_64/corefreqk.c
index e92f194..d1430bd 100644
--- a/x86_64/corefreqk.c
+++ b/x86_64/corefreqk.c
@@ -744,7 +744,7 @@ static const struct {
       { VENDOR_INTEL , __builtin_strlen(VENDOR_INTEL) , CRC_INTEL,  BARE_METAL},
       { VENDOR_AMD   , __builtin_strlen(VENDOR_AMD)   , CRC_AMD  ,  BARE_METAL},
       { VENDOR_HYGON , __builtin_strlen(VENDOR_HYGON) , CRC_HYGON,  BARE_METAL},
-      {VENDOR_ZHAOXIN, __builtin_strlen(VENDOR_ZHAOXIN),CRC_INTEL,  BARE_METAL},
+      // {VENDOR_ZHAOXIN, __builtin_strlen(VENDOR_ZHAOXIN),CRC_INTEL,  BARE_METAL},
       {	VENDOR_KVM   , __builtin_strlen(VENDOR_KVM)   , CRC_KVM  ,  HYPERV_KVM},
       { VENDOR_VBOX  , __builtin_strlen(VENDOR_VBOX)  , CRC_VBOX , HYPERV_VBOX},
       { VENDOR_KBOX  , __builtin_strlen(VENDOR_KBOX)  , CRC_KBOX , HYPERV_KBOX},
@@ -2206,7 +2206,7 @@ static void Map_AMD_Topology(void *arg)
 	case AMD_Zen5_STX:
 	case AMD_Zen5_KRK:
 	case AMD_Zen5_STXH:
-	case AMD_Zen5_GRP:
+	/* case AMD_Zen5_GRP: */
 		CPU_Complex = false;
 		fallthrough;
 	/* Zen CPU Complex */
@@ -2225,7 +2225,7 @@ static void Map_AMD_Topology(void *arg)
 	case AMD_Zen5_Eldora:
 	case AMD_Zen5_Turin:
 	case AMD_Zen5_Turin_Dense:
-	case AMD_Family_1Ah_01h:
+	/* case AMD_Family_1Ah_01h: */
 	case AMD_Zen5_SHP:
 	case AMD_Family_17h:
 	case Hygon_Family_18h:
